src/calendar/__tests__/calendar.test.tsx

```diff
@@ -42,13 +44,19 @@ function getDayText(wrapper: CalendarWrapper, row: number, col: number) {
   return wrapper.findDateAt(row, col).findByClassName(styles['date-inner'])!.getElement().textContent;
 }
 
-describe('Calendar', () => {
-  test('check a11y', async () => {
-    const { container } = renderCalendar();
-    await expect(container).toValidateA11y();
-  });
+test('check a11y', async () => {
+  const { container } = renderCalendar();
+  await expect(container).toValidateA11y();
 });
 
+test.each(range(0, 11).map(month => addMonths(new Date('2025-01-01'), month).toISOString().split('T')[0]))(
```

Comment:
Not critical, but readability could be improved by moving the range to a named variable. For example:
```suggestion
const eachMonthOfTheYear = range(0, 11).map(month => addMonths(new Date('2025-01-01'), month).toISOString().split('T')[0]);
test.each(eachMonthOfTheYear)(
```